config/helmchart/cert-manager-ca-injection.yaml

```diff
@@ -1,4 +1,4 @@
 - op: add
   path: /metadata/annotations
   value:
-    cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
\ No newline at end of file
+    cert-manager.io/inject-ca-from: "{{ .Release.Namespace }}/webhook-server-cert"
```

Comment:
we could but it would not improve the chart functionality because it would just allow the change the name, not to have the cert supplied with a mechanism different from cert-manager. That is the only approach supported for the helm chart. so overall I don't think it's worth it at this point.